### ISSUE TITLE
Improve Design of Text on Badge (size, boldness)

### ISSUE THREAD

**Issue by mariobehling** [fossasia/badgeyay#595]
Please change as follows:
* Make "designation" (Organizer, Speaker etc.) defined in the second column bold and in capitals and on top
* Make name of person larger and bold in the middle
![badges-organisers-a5](https://user-images.githubusercontent.com/1583873/37416615-35915188-27e9-11e8-9aba-97721548cb82.jpg)
* Make organization smaller and normal below the name
* Move Nick or handle with "@" towards the bottom.

![badges-organisers-a5](https://user-images.githubusercontent.com/1583873/37417176-8f93e3ca-27ea-11e8-95de-fc2350b53299.jpg)

**Comment by rupav** [comment]
claiming @mariobehling , you can expect a PR by friday evening.

**Comment by rupav** [comment]
Update: @mariobehling  I have to ungroup those `text`, align them as per requirements, but there is `a black` shade on each badge fixed, which will not allow me to use background and your changes simultaneously. 
![2018-03-17](https://user-images.githubusercontent.com/20624380/37539724-81e2cc78-297a-11e8-8ab2-c938ae417444.png)
![2018-03-17 1](https://user-images.githubusercontent.com/20624380/37539825-daf5ebb0-297a-11e8-851c-2ae5fc35eb9b.png)

Any other approach to tackle this?
@vaibhavsingh97  @S2606 @niccokunzmann

**Comment by sagar-kalra** [comment]
after regrouping, have you tried generating badges?  @rupav

**Comment by rupav** [comment]
Actually this is the original layout. I ungrouped the text to show the background( which shows how the original background is ). I haven't print, because it will not generate badges, python scripts we are using are not compatible yet to print with such ungrouped text.

**Comment by abishekvashok** [comment]
@ruoav i reckon there are other svg files without text.

**Comment by rupav** [comment]
Yes there are,(if you are telling about sample_background svgs), I actually made them in one of the issues, I need to group them together to recreate the page layout (as shown above ) . I was asking if there is a better approach than this where I need to start from beginning.

**Comment by rupav** [comment]
Update: @abishekvashok @sagar-kalra , I am able to move the text away in this svg, https://github.com/fossasia/badgeyay/blob/development/backend/badges/8BadgesOnA3.svg . I will make use of it do the desired changes post 27th march.

**Comment by sagar-kalra** [comment]
@rupav Still working on it?

**Comment by rupav** [comment]
yes, just busy with college stuff.

**Comment by ParthS007** [comment]
@sagar-kalra  I think we should first work on generating badges via csv as it is important before working on enhancement. We should have the basic webapp in working conditions for all the desired inputs.

Your views @gabru-md  @vaibhavsingh97

**Comment by gabru-md** [comment]
Yes. Definitely correct @ParthS007.
Our priority should be generating the badges and then should move on to changing the badge scheme.

**Comment by vaibhavsingh97** [comment]
Yes, me too agree with @ParthS007.

**Comment by sagar-kalra** [comment]
@ParthS007 sure

**Comment by sk9331657** [comment]
@ParthS007  Can I work on this ??
